app/src/main/java/com/gathering/android/auth/password/ForgetPasswordViewModel.kt

```diff
@@ -39,4 +63,11 @@ class ForgetPasswordViewModel @Inject constructor(
         val emailPattern = "[a-zA-Z0-9._-]+@[a-z]+\\.+[a-z]+"
         return email.matches(emailPattern.toRegex())
     }
+
+    companion object {
+        private const val INVALID_EMAIL_ADDRESS = "INVALID EMAIL ADDRESS"
+        private const val FAILED_TO_SEND_RESET_PASSWORD_LINK = "FAILED TO SEND RESET PASSWORD LINK"
+        private const val RESET_PASS_EMAIL_SENT_SUCCESSFULLY =
+            "RESET PASSWORD EMAIL WAS SEND SUCCESSFULLY"
```

Comment:
strings that end up being shown to user should go to resources. you dont need to change it in this PR though. create another ticket for extracting such strings.